cmd/git-go-patch/README.md

```diff
@@ -23,6 +23,17 @@ go install github.com/microsoft/go-infra/cmd/git-go-patch@latest
 > [!NOTE]
 > Make sure `git-go-patch` is accessible in your shell's `PATH` variable. You may need to add `$GOPATH/bin` to your `PATH`. Use `go env GOPATH` to locate it.
 
+## Cross-Platform Shell Support
+
+The interactive shell feature (`rebase -shell`) works on both Unix and Windows:
+
+* **Unix/Linux/macOS**: Uses your `$SHELL` environment variable, defaults to `/bin/bash`
+* **Windows**: Uses your `%COMSPEC%` environment variable, defaults to `cmd.exe`
+
+The shell prompt is automatically updated to show `(git-go-patch)` to indicate you're in the interactive mode:
+* Unix shells: Updates the `PS1` environment variable
+* Windows Command Prompt: Updates the `PROMPT` environment variable
```

Comment:
Nothing for PowerShell? (I think that's probably reasonable--PowerShell is way more complicated about it--but worth mentioning.)